client/site-profiler/components/domain-information/index.tsx

```diff
@@ -54,15 +55,15 @@ export default function DomainInformation( props: Props ) {
 	};
 
 	const formatDateTime = ( date?: string | string[] ) => {
-		if ( Array.isArray( date ) && date.length > 0 ) {
-			date = date[ 0 ];
-		}
-
-		const res = moment.utc( date );
+		const res = moment.utc( normalizeWhoisField( date ) );
 
 		return res.isValid() ? res.format( momentFormat ) : '';
 	};
 
+	const formatRegistrar = ( registrar?: string | string[] ) => {
+		return normalizeWhoisField( registrar );
+	};
+
```

Comment:
Do we even need `formatRegistrar` function when we already have `normalizeWhoisField`?

Reply:
Thanks, fixed.